core/bug_api.php

```diff
@@ -631,6 +637,8 @@ function update( $p_update_extended = false, $p_bypass_mail = false ) {
 
 		$t_old_data = bug_get( $this->id, true );
 
+		event_signal( 'EVENT_UPDATE_BUG_DATA', $this, $t_old_data );
```

Comment:
Should we call again self::validate() after the plugins call?  If they can change the data.